libs/ui-components/src/components/Repository/CreateRepository/CreateRepository.tsx

```diff
@@ -108,6 +107,14 @@ const CreateRepository = () => {
             {repositoryId ? t('Edit repository') : t('Create repository')}
           </Title>
         </StackItem>
+        {rsError && (
+          <Alert isInline variant="warning" title={t('Failed to retrieve the resource syncs')}>
+            {t(
+              'We could not verify if this repository contains resource syncs. Changes to resource syncs are not available at the moment for this reason.',
```

Comment:
Im not very happy with the msg. How about 
```suggestion
              'We could not verify if this repository contains resource syncs. Changes to resource syncs are disabled.',
```

Can we also add a `try again` action ?

Reply:
- I thought of saying "disabled", but since I hid the section completely, it felt that could be confusing.
Would it be better to instead showing the "Use RS" checkbox as disabled and changing the message?

- I will add the "try again" action.

Reply:
I spoke to Nir, and we discussed that it may be better to make the entire Repository read-only when we fail to fetch the RSs, because if they existed, they could be affected by a change in the repository URL, etc.